deepchem/utils/dft_utils/datastruct.py

```diff
@@ -1,14 +0,0 @@
-"""
-Density Functional Theory Data Structure Utilities
-"""
-from typing import Union, TypeVar
```

Comment:
Why are we deleting this file @GreatRSingh ?

Reply:
Everything was working in my local setup.
But, module not found error was happening in the CI.
keeping it inside a folder solved this issue so, I did that,